README.md

```diff
@@ -1,11 +1,11 @@
-# CSI Driver for Dell EMC PowerFlex
+# CSI Driver for Dell PowerFlex
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/dell/csi-vxflexos?style=flat-square)](https://goreportcard.com/report/github.com/dell/csi-vxflexos)
 [![License](https://img.shields.io/github/license/dell/csi-vxflexos?style=flat-square&color=blue&label=License)](https://github.com/dell/csi-vxflexos/blob/master/LICENSE)
 [![Docker](https://img.shields.io/docker/pulls/dellemc/csi-vxflexos.svg?logo=docker&style=flat-square&label=Pulls)](https://hub.docker.com/r/dellemc/csi-vxflexos)
 [![Last Release](https://img.shields.io/github/v/release/dell/csi-vxflexos?label=Latest&style=flat-square&logo=go)](https://github.com/dell/csi-vxflexos/releases)
 
-**Repository for CSI Driver for Dell EMC PowerFlex**
+**Repository for CSI Driver for Dell PowerFlex**
 
 ## Description
 CSI Driver for PowerFlex is part of the [CSM (Container Storage Modules)](https://github.com/dell/csm) open-source suite of Kubernetes storage enablers for Dell EMC products. CSI Driver for PowerFlex is a Container Storage Interface (CSI) driver that provides support for provisioning persistent storage using Dell EMC PowerFlex storage array. 
```

Comment:
Should this line say "Dell products" instead of "Dell EMC products? "
Line below also mentions  "Dell EMC PowerFlex"